change/@ni-nimble-components-e878875a-13bb-474b-b54e-76adb87bc3fa.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Refinements to table selection when using SHIFT",
+  "packageName": "@ni/nimble-components",
+  "email": "20542556+mollykreis@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Could we add a note to the keyboard navigation section of the table storybook to mention these features? I don't think we need to be too detailed in describing the exact behavior but we could at least mention that mutiselect is possible via clicking with Shift and Ctrl.

If we do this we might consider renaming that section to "Keyboard interaction" or just "Interactions"

Reply:
I added a few sentences under "Accessibility" >> "Selecting multiple rows" that's a sibling of the existing "Accessibility" >> "Keyboard navigation" section.